packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js

```diff
@@ -27,6 +27,18 @@ moduleFor('Helpers test: custom helpers', class extends RenderingTest {
     this.assertText('hello | hello world');
   }
 
+  ['@test it does not resolve helpers with a `.` (period)']() {
+    this.registerHelper('hello.world', () => 'hello world');
+
+    this.render('{{hello.world}}{{hello.world foo="bar"}}');
+
+    this.assertText('');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('');
+  }
```

Comment:
Should probably check that it actually worked (by setting "context.hello.world", doing the I-N-U-R)